modules/basic/ds/hashmap.h

```diff
@@ -217,6 +225,262 @@ class HashmapBuilder : public HashmapBaseBuilder<K, V, H, E> {
   std::shared_ptr<Blob> data_buffer_;
 };
 
+template <typename K, typename V>
+class PerfectHashmapBuilder : public PerfectHashmapBaseBuilder<K, V> {
+ public:
+  typedef boomphf::SingleHashFunctor<K> hasher_t;
+
+  explicit PerfectHashmapBuilder(Client& client)
+      : PerfectHashmapBaseBuilder<K, V>(client) {}
+
+  /**
+   * @brief Get the mapping value of the given key.
+   *
+   */
+  inline V& operator[](const K& key) {
+    LOG(INFO) << __func__ << " is not finished yet.";
+    return V(0);
+  }
+
+  /**
+   * @brief Get the mapping value of the given key.
+   *
+   */
+  inline V& operator[](K&& key) {
+    LOG(INFO) << __func__ << " is not finished yet.";
+    return V(0);
+  }
```

Comment:
Remove these method if they cannot be implemented, or implements them to resolve those todos.

Reply:
Done